test/LightAccount.t.sol

```diff
@@ -23,6 +23,8 @@ contract LightAccountTest is Test {
     uint256 public constant EOA_PRIVATE_KEY = 1;
     address payable public constant BENEFICIARY = payable(address(0xbe9ef1c1a2ee));
     bytes32 internal constant _MESSAGE_TYPEHASH = keccak256("LightAccountMessage(bytes message)");
+    address public factoryOwner = 0xDdF32240B4ca3184De7EC8f0D5Aba27dEc8B7A5C;
```

Comment:
Ah, I kinda see why this is in the `.env.example`. Personally, this is why I think it'd be better for the "get initcode hash" logic to be in a script, rather than a test, so it's more ergonomic to load from `.env` (aka tests will pass without having to put something into `.env`).

But I'm OK to keep this in for now and handle later.

Reply:
If the only reason this was in the test was to get the init code hash, I'll remove it. I was wondering if that had been put in the test so that we'd know that we would get a hard warning that we need to deploy new contracts.